testing/builtin_test.py

```diff
@@ -28,6 +28,19 @@
 assert abs(-3.4) == 3.4
 assert isinstance(abs(-3.4), float)
 
+assert abs(complex(0, 0)) == 0.0
```

Comment:
At least one of these assertions is triggering. I verified that they all pass in CPython so it's likely something Go specific.